Pr 4 of 4: Add inline comments where users need to replace code (#121)
AlexsLemonade/refinebio-examples: validate-differential-expression/gene_DE_validate.Rmd

```{r}
# Load in the summary statistics from limma run
author.full.results <- readr::read_tsv(file.path(results_dir, 

>                                                  "GSE37382_limma_results.tsv" # Replace with name of `limma` results file generated in `author_processed_DE.Rmd`

Reviewer: ```suggestion
                                                 "GSE37382_limma_results.tsv" # Replace with name of your own differential expression results file you've generated that you'd like to compare to refinebio data
```
We should make it more clear that it doesn't necessarily need to be this file. The idea of this example is that people can provide their own limma results file to compare with refinebio results. (the `author_processed_DE.Rmd` notebook is just a set up notebook that we used as an example of a file you could compare to, but ideally the user would have some sort of file like this already themselves). 

This being said, we should add more comments at places where metadata or column names for `author.full.results` might differ when our user uploads their own file. Just so they have more heads up where things are likely not to work for every file. (We can't guarantee their limma results file will look the same as ours).

# Add annotation to imported author limma results 
author.full.results <- author.full.results %>%

>   # You may need to replace "affy_probe_ids" with the name of the column that        # contains the probe IDs in your data 

Reviewer: ```suggestion
  # You may need to replace "affy_probe_ids" with the name of the column that        
  # contains the probe IDs in your data 
```

  dplyr::inner_join(probe.conversion, by = c("affy_probe_ids" = "PROBEID")) %>%
  dplyr::mutate("ENSEMBL" = as.factor(ENSEMBL)) %>% 
  # Determine direction of fold change. 
  # Note up-directed genes will be TRUE, down-genes will be FALSE

>   # You may need to replace <logFC> with the name of the column that contains the    # log fold change values in your data 

Reviewer: ```suggestion
  # You may need to replace "logFC" with the name of the column that contains the    
  # log fold change values in your data 
```

```{r}
# Summarize the direction of significant probes for each gene
direction.summary <- author.full.results %>% 

>   # Keep only significant at 0.05 probes -- you may need to replace <adj.P.Val>      # with the name of the column that contains the adjusted P values for your data

Reviewer: ```suggestion
  # Keep only significant at 0.05 probes -- you may need to replace "adj.P.Val" with the corresponding column name in your data that contains the adjusted P values. 
```
